deploy/deploy.go

```diff
@@ -510,6 +510,14 @@ func walkRemote(ctx context.Context, bucket *blob.Bucket) (map[string]*blob.List
 		if err != nil {
 			return nil, err
 		}
+		// Check include/exclude matchers.
+		slashpath := filepath.ToSlash(obj.Key)
```

Comment:
I don't think that this `ToSlash` is necessary; gocloud blob keys should already be slashified. Note that below, the object is added to `retval` under the key `obj.Key`, and that's compared directly to the local file path; that wouldn't work if my assertion was false.

Reply:
Thanks yes you are correct without ToSlash it works too